File: Bulgarian Olympiad 2011/BNOI 2011 - Round 3 - Group C (7-8 grade)/Clock/Clock.cs
using System;

class Clock
{
    static void Main()
    {
        string[] input = Console.ReadLine().Split(':');
        int min = int.Parse(input[1]);
        double alpha = 360 * Math.Abs(min / 60d - (int.Parse(input[0]) / 12d + min / (60 * 12d)));
        Console.WriteLine("{0:F3}", (alpha <= 180) ? alpha : 360 - alpha);
    }
}

File: BNOI - Group C/Clock/Clock.js
function solve(args) {
    var input = args[0].split(':');
    var min = parseInt(input[1]);
    var alpha = 360 * Math.abs(min / 60 - (parseInt(input[0]) / 12 + min / (60 * 12)));

    return ((alpha <= 180) ? alpha : 360 - alpha).toFixed(3);
}